src/GAP/gap_to_oscar.jl

```diff
@@ -204,12 +204,12 @@ function matrices_over_field(gapmats::GapObj)
     GAPWrap.IsList(gapmats) || throw(ArgumentError("gapmats is not a GAP list"))
     GAPWrap.IsEmpty(gapmats) && throw(ArgumentError("gapmats is empty"))
 
-    if GAP.Globals.IsFFECollCollColl(gapmats) ||
-       GAP.Globals.IsCyclotomicCollCollColl(gapmats)
+    if GAPWrap.IsFFECollCollColl(gapmats) ||
+       GAPWrap.IsCyclotomicCollCollColl(gapmats)
       # If the entries of the matrices are FFEs or cyclotomics then
       # we construct the field generated by all entries.
       gapF = GAP.Globals.FieldOfMatrixList(gapmats)
-    elseif GAP.Globals.IsAlgebraicElementCollCollColl(gapmats) &&
+    elseif GAPWrap.IsAlgebraicElementCollCollColl(gapmats) &&
```

Comment:
@ThomasBreuer it seems we also have no CI test for this conversion case? Can you add one here, too?

Reply:
Yes.